### PR TITLE
Attempt at addressing the throttle and brake issue

### PR DESCRIPTION
**Pull request by jfjensen**
I've spent the day trying to tweak the PID and low pass filter parameters.
What I have now is the best I can come up with.
If you guys want to test the current settings, I encourage you to do the following:
When running the Sim, start `rqt_plot` in a new terminal and add either of:
* `/velocity_error/data` (this is a new topic I added to view the velocity error)
* `/vehicle/throttle_cmd/pedal_cmd` and `/vehicle/brake_cmd/pedal_cmd`
... and then view the plot.

### CODE REVIEW

**Comment by eljefec** [comment]
Can you share the sim plot?

**Comment by eljefec** [comment]
Have you tried testing this on the successful ROS bag? I assume Justin has some code or tips for generating the plots that he shared.

**Review by eljefec — COMMENTED** [review]
I added a couple of comments.

File: ros/src/twist_controller/dbw_node.py [hunk 1]
```diff
@@ -72,13 +72,19 @@ def __init__(self):
...
+        # self.controller = Controller(kp=0.1, ki=0.0, kd=0.05, vc=vc) # fits expectation better
+        # self.controller = Controller(kp=0.1, ki=0.0, kd=0.03, vc=vc) # fits expectation even better
+        # self.controller = Controller(kp=0.1, ki=0.0, kd=0.02, vc=vc) # fits expectation even better - car overshoots
+        # self.controller = Controller(kp=0.1, ki=0.0, kd=0.025, vc=vc) # good
+        # self.controller = Controller(kp=0.15, ki=0.001, kd=0.025, vc=vc) # 
+        # self.controller = Controller(kp=0.9, ki=0.0, kd=0.0, vc=vc) # good
+        # self.controller = Controller(kp=1.0, ki=0.0, kd=0.0, vc=vc) #
+        # self.controller = Controller(kp=1.6, ki=0.0, kd=0.01, vc=vc) # good
+        # self.controller = Controller(kp=1.6, ki=0.0, kd=0.04, vc=vc) # good ++++++++++++++
+        # self.controller = Controller(kp=1.6, ki=0.00005, kd=0.04, vc=vc) # good ++++++++++++
```
Comment (mwolfram) [inline ros/src/twist_controller/dbw_node.py:84]: These lines call for dynamic reconfigure: http://wiki.ros.org/dynamic_reconfigure
So if there's enough time, we should integrate it.

**Review by mwolfram — APPROVED** [review]

File: ros/src/twist_controller/dbw_node.py (hunk 1 above)
Comment (jfjensen) [reply ros/src/twist_controller/dbw_node.py:84]: Yes, I agree... if indeed, there's enough time...

**Pull request merged**